### ISSUE TITLE
[CLOSED] Reproducible seeding for dataloaders

### ISSUE THREAD

**Issue by mogwai** [mogwai/pyannote-test#3]
<a href="https://github.com/mogwai"><img src="https://avatars1.githubusercontent.com/u/5440566?v=4" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [mogwai](https://github.com/mogwai)**
_Monday Oct 12, 2020 at 12:06 GMT_
_Originally opened as https://github.com/hbredin/pyannote-audio-v2/issues/3_

----

Ensure the data loaders produce the same items with multiple workers when setting the global seed with pytorch lightning.
